pkg/services/alerting/notifiers/base.go

```diff
@@ -25,7 +26,13 @@ func NewNotifierBase(id int64, isDefault bool, name, notifierType string, model
 	}
 }
 
-func (n *NotifierBase) PassesFilter(rule *alerting.Rule) bool {
+func DefaultShouldNotify(context *alerting.EvalContext) bool {
```

Comment:
I think this should be `defaultShouldNotify` we don't need to expose this outside the notifiers package.